docker/Dockerfile.cpu

```diff
@@ -24,7 +24,20 @@ RUN apt-get install -y libgl1-mesa-glx
 ##############################################################
 FROM base as gcc
 
-RUN apt-get install -y mpich libmpich-dev
+RUN mkdir -p /deps && mkdir -p /opt/openmpi && cd /deps && \
```

Comment:
@mloubout for homogeneity, switching to OpenMPI here

Reply:
I don't mind switching but this is the expensive route, for standard CPU `apt install libopenmpi-dev` should be enough (gotta double check name)

Reply:
but last I checked you would get openmpi 3, this way you get 4 (but I may be wrong)

Reply:
Ok, it's fine

`mkdir -p` lets you do any path so just `mkdir -p /opt/openmpi`

Reply:
missread `deps` for `openmpi` my bad